src/types/oec/okex.ts

```diff
@@ -0,0 +1,294 @@
+
+[rest]
+laddr = "tcp://0.0.0.0:{{RPC_PORT}}"
+
+[p2p]
+laddr = "tcp://0.0.0.0:{{PEER_PORT}}"
+`;
+
+export class OKEX extends Ethereum {
+
+  static versions(client: string, networkType: string): VersionDockerImage[] {
+    client = client || OKEX.clients[0];
+    let versions: VersionDockerImage[];
+    switch (client) {
+      case NodeClient.CORE:
+        versions = [
+          {
+            version: '1.1.4.3',
+            clientVersion: '1.1.4.3',
+            image: 'okexchain/fullnode-mainnet:v1.1.4.3',
+            dataDir: '/root/.exchaind/data',
+            walletDir: '/root/keystore',
+            configDir: '/root/.exchaind/config',
+            networks: [NetworkType.MAINNET],
+            breaking: false,
+            generateRuntimeArgs(data: CryptoNodeData): string {
+              return ' start'
+            },
+          },
+          {
+            version: '1.1.4',
+            clientVersion: '1.1.4',
+            image: 'okexchain/fullnode-mainnet:v1.1.4',
+            dataDir: '/root/.exchaind/data',
+            walletDir: '/root/keystore',
+            configDir: '/root/.exchaind/config',
+            networks: [NetworkType.MAINNET],
+            breaking: false,
+            generateRuntimeArgs(data: CryptoNodeData): string {
+              return ' start'
+            },
+          },
+          {
+            version: '1.1.2',
+            clientVersion: '1.1.2',
+            image: 'okexchain/fullnode-mainnet:v1.1.2',
+            dataDir: '/root/.exchaind/data',
+            walletDir: '/root/keystore',
+            configDir: '/root/.exchaind/config',
+            networks: [NetworkType.MAINNET],
+            breaking: false,
+            generateRuntimeArgs(data: CryptoNodeData): string {
+              return ' start'
+            },
+          },
+        ];
+        break;
+      default:
+        versions = [];
+    }
+    return filterVersionsByNetworkType(networkType, versions);
+  }
+
+  static clients = [
+    NodeClient.CORE,
+  ];
+
+  static nodeTypes = [
+    NodeType.FULL,
+  ];
+
+  static networkTypes = [
+    NetworkType.MAINNET,
+  ];
+
+  static roles = [
+    Role.NODE,
+  ];
+
+  static defaultRPCPort = {
+    [NetworkType.MAINNET]: 8545,
+  };
+
+  static defaultPeerPort = {
+    [NetworkType.MAINNET]: 26656,
+  };
+
+  static defaultCPUs = 4;
+
+  static defaultMem = 8192;
+
+  static generateConfig(client = OKEX.clients[0], network = NetworkType.MAINNET, peerPort = OKEX.defaultPeerPort[NetworkType.MAINNET], rpcPort = OKEX.defaultRPCPort[NetworkType.MAINNET], moniker = 'nodepilot'): string {
+    switch (client) {
+      case NodeClient.CORE:
+        return coreConfig
+          .replace('{{PEER_PORT}}', peerPort.toString(10))
+          .replace('{{RPC_PORT}}', rpcPort.toString(10))
+          .replace('{{MONIKER}}', moniker)
+          .trim();
+      default:
+        return '';
+    }
+  }
+  
+  static async generateGenesis(configDir: string, newtork: string) {
```

Comment:
Please replace this with a static genesis like we use with [Pocket](https://github.com/decentralized-authority/node-launcher/blob/dev/src/types/pocket/genesis.ts). We do not want Node Launcher to be dependent on any external sources or APIs outside of the blockchain network itself.

Reply:
I was unable to follow the exact style used in the example you showed due to some character encoding issues. But I was able to get it to work by downloading and saving a raw local copy of the genesis file in the oec directory and then copying this over when mounting the docker drives.